options/locale/locale_en-US.ini

```diff
@@ -1752,6 +1752,12 @@ issues.review.hide_resolved = Hide resolved
 issues.review.resolve_conversation = Resolve conversation
 issues.review.un_resolve_conversation = Unresolve conversation
 issues.review.resolved_by = marked this conversation as resolved
+issues.review.comment = Comment
+issues.review.official = Approved
+issues.review.requested = Review pending
+issues.review.rejected = Changes requested
+issues.review.stale = Updated since approval
+issues.review.unofficial = Unofficial approval
```

Comment:
If this is only for the tooltip, maybe we can have more descriptions like https://github.com/go-gitea/gitea/pull/31924#issuecomment-2315496446 mentioned. `grey` means uncounted approval, the new name maybe better than `unofficial` because official reviewer's previous reviews will be uncounted.

Reply:
Text updated 👍